ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -21,8 +21,10 @@
 log = get_logger(__name__)
 
 
-def enable_appsec_rc(tracer):
-    # type: (Tracer) -> None
+def enable_appsec_rc():
+    # type: () -> None
+    from ddtrace import tracer
```

Comment:
this is to avoid circular imports right?

if so, can we leave a comment so someone doesn't come along later and hoist it up to the top of the file?